Sources/MapboxCoreNavigation/LegacyRouteController.swift

```diff
@@ -286,7 +286,7 @@ open class LegacyRouteController: NSObject, Router, InternalRouter, CLLocationMa
     private func checkForUpdates() {
         #if TARGET_IPHONE_SIMULATOR
         guard (NSClassFromString("XCTestCase") == nil) else { return } // Short-circuit when running unit tests
-        guard let version = Bundle.string(forMapboxCoreNavigationInfoDictionaryKey: "CFBundleShortVersionString") else { return }
```

Comment:
Should we patch up `Bundle.string(forMapboxCoreNavigationInfoDictionaryKey:)` and `Bundle.string(forMapboxNavigationInfoDictionaryKey:)` to report the same information as VersionProvider?

Reply:
Patched these methods, and now they look into `MBXInfo.plist` first, and only then into `Info.plist`. But in general, these methods shouldn't be used to resolve the current SDK version, because they don't handle all cases. Instead, now we should use my new property `Bundle.navigationSDKVersion`.